### DISCUSSION TITLE
error 500

### DISCUSSION THREAD

**Discussion by denchellaa** [community/community#33139] [Actions] [answered]
i have error 500 , what should i do?

**Comment by radar-blip** [comment, marked as answer]
Sign into your profile page, should work.
`https://github.com/username`

**Comment by liquidsmok3** [comment]
having this same problems for a couple weeks now, and signing in to my profile doesn't fix it.

**Reply by wxy1203** [reply]
me too, still cannot access my profile, what is the problem here?

**Reply by Sahill17** [reply]
Same here :(

**Reply by jpmaniqis** [reply]
> having this same problems for a couple weeks now, and signing in to my profile doesn't fix it.

**Comment by AirditSoftware** [comment]
Getting 500 error while trying click on pay now.

**Reply by AirditSoftware** [reply]
![Uploading image.png…]()

**Comment by robsanBayisa** [comment]
HI

**Comment by robsanBayisa** [comment]
ERROR
